src/components/FlagSelect.tsx

```diff
@@ -1,7 +1,7 @@
 import { useEffect, useState } from 'react';
 import { makeStyles } from '@material-ui/core/styles';
-import Australia from 'flags/Australia.png';
-import america from 'flags/america.png';
+import Australia from 'assets/Australia.png';
+import america from 'assets/america.png';
```

Comment:
i would suggest you to create images folder inside assets and put all your images there